### ISSUE TITLE
Fiddle Very Slow on MacOS BigSur

### ISSUE THREAD

**Issue by ikkisoft** [electron/fiddle#581]
On the latest version 0.20.0 (11.1.1), it takes several seconds (10-20 seconds) to bootstrap the app once I click on the "Play" button.

This is on MacOS BigSur - latest MacPro 32 GB. Have not tried on different platforms.

It's almost not usable for fast prototyping.

**Comment by erickzhao** [comment]
@ikkisoft do older versions work better? Was this a recent regression?

**Comment by ikkisoft** [comment]
Older versions were definitely faster (a couple of seconds to start) but I've updated from Catalina to Big Sur AND updated Fiddle. Let me try on a few VMs / versions, and let you know more details.

**Comment by ikkisoft** [comment]
I've tested on both Windows 10, Mac OS Catalina and Mac Big Sur using new VMs. No problems after a fresh install.

In my case, I believe it was caused by the OS upgrade and multiple Fiddle upgrades. 
**Deleting ~/Library/Application\ Support/Electron\ Fiddle and /Applications/Electron\ Fiddle.app, then installing the latest release fixed the issue.**

I assume there is nothing to do, so closing the issue
